src/freenet/support/PrioritizedTicker.java

```diff
@@ -267,54 +252,81 @@ public Executor getExecutor() {
 
 	public int queuedJobs() {
 		synchronized(timedJobsByTime) {
-			return timedJobsByTime.size();
+			return timedJobsQueued.size();
 		}
 	}
 
+    public int queuedJobsUniqueTimes() {
```

Comment:
I don't think this should be part of the public API.

Do we need this outside of unit tests? If not, please make it package-private.